frontend/src/components/templates/HomeLayout/SidebarLayout.js

```diff
@@ -38,7 +37,17 @@ export const SidebarLayout = ({sidebarComponents}) => {
                         />
                     } />
                 }
-                else{
+                else if(component.isKeypair) {
```

Comment:
This doesn't look like a part of this stub. Looks like this is meant to actually get added to the UI.

Reply:
Yes, we need it to be able to reuse the same component